zebrad/src/commands/start/sync.rs

```diff
@@ -269,65 +314,31 @@ where
     }
 
     /// Queue downloads for each block that isn't currently known to our node
-    #[instrument(skip(self, hashes))]
     async fn request_blocks(&mut self, hashes: Vec<BlockHeaderHash>) -> Result<(), Report> {
         tracing::debug!(hashes.len = hashes.len(), "requesting blocks");
-        for chunk in hashes.chunks(10usize) {
-            let set = chunk.iter().cloned().collect();
-
-            let request = self
-                .retry_peer_set
-                .ready_and()
-                .await
-                .map_err(|e| eyre!(e))?
-                .call(zn::Request::BlocksByHash(set));
-
-            let verifier = self.verifier.clone();
-
-            let _ = tokio::spawn(
-                async move {
-                    // XXX for some reason the tracing filter
-                    // filter = 'info,[sync]=debug'
-                    // does not pick this up, even though this future is instrumented
-                    // with the current span below.  However, fixing it immediately
-                    // isn't critical because this code needs to be changed to propagate
-                    // backpressure to the syncer.
-                    tracing::debug!("test");
-                    let result_fut = async move {
-                        let mut handles = FuturesUnordered::new();
-                        let resp = request.await?;
-
-                        if let zn::Response::Blocks(blocks) = resp {
-                            debug!(count = blocks.len(), "received blocks");
-
-                            for block in blocks {
-                                let mut verifier = verifier.clone();
-                                let handle = tokio::spawn(async move {
-                                    verifier.ready_and().await?.call(block).await
-                                });
-                                handles.push(handle);
-                            }
-                        } else {
-                            debug!(?resp, "unexpected response");
-                        }
-
-                        while let Some(res) = handles.next().await {
-                            let _hash = res??;
-                        }
-
-                        Ok::<_, Error>(())
+        for hash in hashes.into_iter() {
+            let mut retry_peer_set = self.block_network.clone();
+            let mut verifier = self.verifier.clone();
+            let span = tracing::info_span!("block_fetch_verify", hash = ?hash);
```

Comment:
```suggestion
            let span = tracing::info_span!("block_fetch_verify", ?hash);
```

the identifier is redundant in this case